integration_tests/handler/contest_test.go

```diff
@@ -500,6 +505,36 @@ func TestEditContestTeam(t *testing.T) {
 			},
 			nil,
 		},
+		"204 without change": {
+			http.StatusNoContent,
+			mockdata.ContestID1(),
+			mockdata.ContestTeamID1(),
```

Comment:
```suggestion
			mockdata.ContestTeamID2(),
```
にして動くならそっちにしてほしいです
204が2つあってどっちも同じチームを変更すると競合が起きそうなので